R/tt_export.R

```diff
@@ -504,86 +504,3 @@ export_as_pdf <- function(tt,
     }
      list(file = file, npages = npages, exceeds_width = exceeds_width, exceeds_height = exceeds_height, lpp = lpp, cpp = cpp)
 }
-
-.margin_lines_to_in <- function(margins, font_size, font_family) {
```

Comment:
hi @gmbecker , this was part of the migration right?

Reply:
Yes, its no longer needed because we are now interpreting margins as inches values directly.